lib/combo.coffee

```diff
@@ -482,23 +514,30 @@
         if rawValue isnt  "" and !@hasSelection()
           htmls.push("<li class='unmatched-raw-value'>#{rawValue}</li>")
 
-      for item, index in items
-        continue if @onlyShowEnabled and not @itemEnabled(item)
-        continue if not _.all filters, (filter) -> filter.predicate filter.getter(item)
-        htmls.push @renderItem item, index, filters
-
+      Array.prototype.push.apply(htmls, @renderItemList(items, filters))
+      Array.prototype.push.apply(htmls, @renderItemList(secondaryItems, filters, 'secondary-source', items.length))   
+      
       if htmls.length
         @list.html htmls.join('')
+        # append classname "first" to the first secondary-source, it is style related
+        @list.find('.secondary-source').first().addClass('first')
       else
         @list.html "<li class='disabled' data-combo-id='emptylist-item'>#{@emptyListText}</li>"
+        
+    renderItemList: (items, filters, className = '', itemOffset = 0) =>

```

Comment:
Tænk lige over den her, jeg kunne godt tænke mig at slippe for offset.
hvad med renderItemIntoList som tager imod htmls og sætter id som matcher længden og appender?

Derudover tror jeg offset er forkert, kommer id'et ikke til at være off-by-one i følgende tilfælde:
[rawItem, a, secondary-b]. Der ser ud til at combo-id bliver hhv. 0 og 1 for 'a' og 'secondary-b', men hvis vi bruger det som index får vi en rawItem, a tilbage?

du må meget gerne lave en test der spotter fejlen - hvis jeg har ret.